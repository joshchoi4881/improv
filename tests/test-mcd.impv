func int main() {
  note[] arr;
  arr = [<1, "qr">, <1, "qr">, <1, "qr">, <4, "qr">,
    <0, "qr">, <0, "qr">, <4, "hf">, <3, "qr">,
    <3, "qr">, <2, "qr">, <2, "qr">, <1, "hf">];
  render(arr, "demo/mcd.mid", 13, 120);
  return 0;
}