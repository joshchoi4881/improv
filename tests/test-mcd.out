finished creating demo/mcd.mid!
